isis/CMakeLists.txt

```diff
@@ -490,6 +497,13 @@ add_custom_target(incs ALL COMMAND ${CMAKE_COMMAND} -E copy_if_different
   ${CMAKE_SOURCE_DIR}/src/*/apps/*/*.h ${CMAKE_BINARY_DIR}/inc)
 add_dependencies(isis3 incs)
 
+# Add custom build target to copy modified translation files to the build/appdata/translations
+# directory. (As above - may need to be updated)
+add_custom_target(appdata ALL COMMAND ${CMAKE_COMMAND} -E copy_if_different
+  ${CMAKE_SOURCE_DIR}/appdata/serialnumbers/*.trn ${CMAKE_SOURCE_DIR}/src/*/apps/*/*.trn
+  ${CMAKE_BINARY_DIR}/appdata/translations)
```

Comment:
Will this copy the base translation files that will be in isis/appdata/translations to the build dir?

Reply:
That isn't in this PR and is a very good point! I was planning to expand this to include the other directories (including everything that needs to be migrated from base) in a future PR, unless not having it in here is holding anyone up?

Reply:
Oh gotcha. I think it's fine to have it in a PR later with the rest of the base stuff, if everyone else thinks so.